examples/src/main/java/io/github/trimax/examples/utils/VectorUtil.java

```diff
@@ -0,0 +1,21 @@
+package io.github.trimax.examples.utils;
+
+import lombok.experimental.UtilityClass;
+import org.joml.Vector3f;
+
+import java.util.Random;
+
+@UtilityClass
+public final class VectorUtil {
+    public Vector3f createRandomVector3(final float min, final float max) {
+        final Random random = new Random();
+        random.setSeed(System.nanoTime());
```

Comment:
Could be simplified. Just put `System.nanoTime()` to the constructor of `Random` class